week2/materials/homework/assignments.md

```diff
@@ -4,27 +4,28 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/soujanya-week2-homework?region=us-east-1&tab=objects`
```

Comment:
I am missing the public url so I can look at your website :)

Reply:
@bytewiz Sure i will update soon.Thank you :)